src/commands/performance/bandwidth_throttle/update.ts

```diff
@@ -0,0 +1,56 @@
+import * as fs from 'fs';
+import { runCommand } from '../../../util/command';
+import { writeConfiguration } from './util';
+import { Performance } from '../../../sdk';
+
+export = {
+	command: 'update',
+	description: 'Update bandwidth throttle',
+	builder: yargs => {
+		yargs.options({
+			'upload': {
+				alias: 'u',
+				desc: 'The throttle options for uploads.',
+				type: 'string'
+			},
+			'upload-file': {
+				alias: 'u-file',
```

Comment:
alias should be `'uf'` so that it can be used as a shorthand: node bin/skysync.js performance bandwidth_throttle update -uf upload.json

Reply:
changed